.env

```diff
@@ -0,0 +1,2 @@
+PORT=3001
+DATABASE_URL=mongodb://localhost:27017/books
```

Comment:
Feedback here: Please ensure .env is added to .gitignore 👍

Reply:
- Please ensure that node modules is also added to .gitignore as they were pushed up to git. 
- Before merging PR's always try to take a look at `Files Changed` section of PR. This tells you pretty quickly if something is awry :)

-Grab a TA to help with this 👍